SPECS/docbook5-schemas/docbook5-schemas.spec

```diff
@@ -0,0 +1,397 @@
+Summary:        Norman Walsh's schemas (DTD, Relax NG, W3C schema) for Docbook 5.X
+Name:           docbook5-schemas
+Version:        5.1
+Release:        3%{?dist}
+License:        Freely redistributable without restriction
+URL:            https://docbook.org/
+Vendor:         Microsoft
+Distribution:   Mariner
+#Source0:       https://www.docbook.org/xml/%{version}/docbook-v%{version}-os.zip
+Source0:        %{name}-%{version}.zip
+Source1:        https://www.docbook.org/xml/5.0/docbook-5.0.zip
+Source2:        LICENSE
```

Comment:
>C [](start = 18, length = 1)

Where is this file coming from? If there is no licensing information available in the sources, but it's available elsewhere, we should mention where we got that information from. The `LICENSE.PTR` for the `gnu-efi` package is a good example and can be found under `SPECS/gnu-efi/LICENSE.PTR`. In short: in addition to the license info itself, please add a few lines inside the `LICENSE` file explaining why it's added separately and yet still represents the license information for this package.

Reply:
Removed the file, added a '%license docbook-v5.1-os.html' in the files section.